templates/download/server/index.html

```diff
@@ -62,6 +62,34 @@ <h2>Ubuntu Server {{ releases.latest.full_version }}</h2>
   </div>
 </section>
 
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h2>Multipass for instant Ubuntu VMs</h2>
+      <p>With Multipass you can download, configure, and control Ubuntu Server virtual machines with latest updates preinstalled. Set up a mini-cloud on your Linux, Windows, or macOS system.</p>
+
+      <p>
+        <a id="multipass-os-cta" href="https://multipass.run/docs" class="p-button--neutral"><span class="p-link--external">Download</span></a>
+      </p>
+
+      <p>
+        <a href="https://multipass.run" class="p-link--external">Learn more about multipass</a>
+      </p>
+    </div>
+    <div class="col-5 u-vertically-center u-align--center">
```

Comment:
I don't think we want the logo on small screens.

Reply:
Good point